plugins/versionlock.py

```diff
@@ -75,15 +78,21 @@ def sack(self):
                 excl = True
 
             subj = dnf.subject.Subject(pat)
-            solution = subj._get_nevra_solution(self.base.sack, with_nevra=True,
-                                                with_provides=False, with_filenames=False)
+            possible_nevras = list(subj.get_nevra_possibilities(forms=[hawkey.FORM_NEVRA]))
+            if not possible_nevras:
+                logger.info("%s %s", NEVRA_ERROR, pat)
+                continue
+            nevra = possible_nevras[0]
+            pat_query = nevra.to_query(self.base.sack)
 
             if excl:
-                excludes_query = excludes_query.union(solution['query'])
-            elif solution['nevra'] is not None and solution['nevra'].name is not None:
-                locked_names.add(solution['nevra'].name)
-                locked_query = locked_query.union(solution['query'])
-        if locked_query:
+                logger.info("%s %s", APPLY_EXCLUDE, pat)
```

Comment:
It cannot be info, but debug. 
I also would prefer if it will log all excludes or locks at once. See "sudo dnf install -v acpi"